addons/website_sale/controllers/main.py

```diff
@@ -1040,6 +1040,7 @@ def _prepare_checkout_page_values(self, order_sudo, **_kwargs):
             '|',
             ('type', 'in', ['delivery', 'other']),
             ('id', '=', commercial_partner_sudo.id),
+            ('is_pickup_location', '!=', True),
```

Comment:
```suggestion
            ('is_pickup_location', '=', False),
```

Reply:
Did you make sure that mondialrelay/ups/... pickup points are correctly flagged as pickup location ? (I don't know exactly how they work and I don't see any enterprise PR :D)

Reply:
Maybe worth adding a hook and extend it with mondialrelay to exclude those too for now ? (or make a script that set the field to True if `is_mondialrelay` is truthy ?)

Reply:
for mondialrelay they create a partner on the fly, when a customer has chosen a pickup point not on order confirmation. And when a pickup point is chosen it is not shown on the provider  form but shown as an address card in the delivery addresses section, so mondialrelay card does not need to be hidden as they flow is special.

Reply:
Okay, thanks for the explanation :+1: